client/reader/following-manage/connected-subscription-list-item.jsx

```diff
@@ -22,26 +22,35 @@ class ConnectedSubscriptionListItem extends React.Component {
 		feedId: PropTypes.number,
 		siteId: PropTypes.number,
 		onLoad: PropTypes.func,
+		onRender: PropTypes.func,
 		showEmailSettings: PropTypes.bool,
 		showLastUpdatedDate: PropTypes.bool,
 		isFollowing: PropTypes.bool,
 		followSource: PropTypes.string,
+		railcar: PropTypes.object,
 	};
 
 	static defaultProps = {
 		onLoad: noop,
+		onRender: noop,
 		showEmailSettings: true,
 		showLastUpdatedDate: true,
 	};
 
 	componentDidMount() {
 		this.props.onLoad();
+		if ( this.props.railcar ) {
```

Comment:
logically this logic should be flipped.
It should always call `this.props.onRender( this.props )`.